core/payment-driver/gnt/src/service.rs

```diff
@@ -34,28 +36,33 @@ pub async fn subscribe_to_identity_events() -> anyhow::Result<()> {
 
 pub async fn register_in_payment_service() -> anyhow::Result<()> {
     log::debug!("Registering driver in payment service...");
-    let networks = hashmap! {  // TODO: Implement multi-network support
-        DEFAULT_NETWORK.to_string() => payment_srv::Network {
-            default_token: DEFAULT_TOKEN.to_string(),
-            tokens: hashmap! {
-                DEFAULT_TOKEN.to_string() => DEFAULT_PLATFORM.to_string()
-            }
-        }
-    };
     let message = payment_srv::RegisterDriver {
         driver_name: DRIVER_NAME.to_string(),
-        details: payment_srv::DriverDetails {
-            default_network: DEFAULT_NETWORK.to_string(),
-            networks,
-            recv_init_required: false,
-        },
+        details: DRIVER_DETAILS.clone(),
     };
     service(payment_srv::BUS_ID).send(message).await?.unwrap(); // Unwrap on purpose because it's NoError
     log::debug!("Successfully registered driver in payment service.");
 
     Ok(())
 }
 
+async fn fund(
+    _db: DbExecutor,
+    processor: GNTDriverProcessor,
+    _caller: String,
+    msg: Fund,
+) -> Result<String, GenericError> {
+    log::info!("fund: {:?}", msg);
```

Comment:
msg should be logged on debug since its ugly, maybe a neat UX friendly message saying "Preparing to fund address {}..."